openfisca_core/entities.py

```diff
@@ -281,15 +285,15 @@ def init_from_json(self, entities_json):
 
         Entity.init_from_json(self, entities_json)
```

Comment:
Choose between attributes like `entities_json` and calling the simulation object ?  Simulation is called before and it knows the json entities, doesn't it ?